vue/components/ui/organisms/form/form.stories.js

```diff
@@ -47,11 +49,17 @@ storiesOf("Organisms", module)
                                 fields: [
                                     {
                                         value: "username3",
-                                        type: "string"
+                                        type: "text"
                                     },
                                     {
-                                        value: "email3",
-                                        type: "email"
+                                        value: "description",
+                                        type: "text",
+                                        meta: "long"

```

Comment:
Let's use `longtext` in order to be aligned with appier: https://github.com/hivesolutions/appier/blob/0d1b84a90fccd6759b38fc643012e85281ed7d29/src/appier/model.py#L81